src/index.scss

```diff
@@ -37,7 +37,7 @@ a {
 
 input,
 button {
-  outline: none;
+  outline: none !important;
```

Comment:
why are you using `!important`?

Reply:
Because sometimes ```outline: none;``` is being overridden by some other class or id. So, I have added `!important` so that ```outline: none``` cannot be changed.